charts/console/deployment.go

```diff
@@ -62,6 +62,9 @@ func Deployment(dot *helmette.Dot) *appsv1.Deployment {
 	if values.InitContainers.ExtraInitContainers != nil {
 		initContainers = helmette.UnmarshalYamlArray[corev1.Container](helmette.Tpl(*values.InitContainers.ExtraInitContainers, dot))
 	}
+	if initContainers == nil {
```

Comment:
Why has this been added? Shouldn't we be seeing updates to console's test cases?